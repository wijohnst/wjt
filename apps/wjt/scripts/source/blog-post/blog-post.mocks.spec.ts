import { getMockPostContent, mockValues } from './blog-post.mocks';

describe('getMockPostContent', () => {
  test('should be defined', () => {
    expect(getMockPostContent).toBeDefined();
  });

  test('return value should match snapshots', () => {
    expect(getMockPostContent(mockValues[0])).toMatchSnapshot();
  });
});
