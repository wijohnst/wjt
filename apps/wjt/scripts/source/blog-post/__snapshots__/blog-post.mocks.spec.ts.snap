// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getMockPostContent return value should match snapshots 1`] = `
"---
title: Post 1
author: Some Author
slug: post-1
---
# Post 1

This is the first post."
`;
